README.md

```diff
@@ -2,116 +2,27 @@
-      logging = "OVERRIDE"
-      log_configuration = {
-        cloud_watch_log_group_name = "/aws/ecs/aws-ec2"
-      }
-    }
-  }
-
-  fargate_capacity_providers = {
-    FARGATE = {
-      default_capacity_provider_strategy = {
-        weight = 50
-      }
-    }
-    FARGATE_SPOT = {
-      default_capacity_provider_strategy = {
-        weight = 50
-      }
-    }
-  }
-
-  tags = {
-    Environment = "Development"
-    Project     = "EcsEc2"
-  }
-}
-```
-
-### EC2 Autoscaling Capacity Providers
-
-```hcl
-module "ecs" {
-  source = "terraform-aws-modules/ecs/aws"
-
-  cluster_name = "ecs-ec2"
+[![SWUbanner](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner2-direct.svg)](https://github.com/vshymanskyy/StandWithUkraine/blob/main/docs/README.md)
 
-  cluster_configuration = {
-    execute_command_configuration = {
-      logging = "OVERRIDE"
-      log_configuration = {
-        cloud_watch_log_group_name = "/aws/ecs/aws-ec2"
-      }
-    }
-  }
+## Available Features
 
-  autoscaling_capacity_providers = {
-    one = {
-      auto_scaling_group_arn         = "arn:aws:autoscaling:eu-west-1:012345678901:autoScalingGroup:08419a61:autoScalingGroupName/ecs-ec2-one-20220603194933774300000011"
-      managed_termination_protection = "ENABLED"
+- ECS cluster w/ Fargate or EC2 Auto Scaling capacity providers
+- ECS Service w/ task definition, task set, and container definition support
+- Separate sub-modules or integrated module for ECS cluster and service
 
-      managed_scaling = {
-        maximum_scaling_step_size = 5
-        minimum_scaling_step_size = 1
-        status                    = "ENABLED"
-        target_capacity           = 60
-      }
+For more details see the [design doc](https://github.com/terraform-aws-modules/terraform-aws-ecs/blob/master/docs/README.md)
 
-      default_capacity_provider_strategy = {
-        weight = 60
-        base   = 20
-      }
-    }
-    two = {
-      auto_scaling_group_arn         = "arn:aws:autoscaling:eu-west-1:012345678901:autoScalingGroup:08419a61:autoScalingGroupName/ecs-ec2-two-20220603194933774300000022"
-      managed_termination_protection = "ENABLED"
-
-      managed_scaling = {
-        maximum_scaling_step_size = 15
-        minimum_scaling_step_size = 5
-        status                    = "ENABLED"
-        target_capacity           = 90
-      }
+## Usage
 
-      default_capacity_provider_strategy = {
-        weight = 40
-      }
-    }
-  }
+This project supports creating resources through individual sub-modules, or through a single module that creates both the cluster and service resources. See the respective sub-module directory for more details and example usage.
 
-  tags = {
-    Environment = "Development"
-    Project     = "EcsEc2"
-  }
-}
-```
-
-### Fargate & EC2 Autoscaling Capacity Providers
+### Integrated Cluster w/ Services
 
 ```hcl
 module "ecs" {
-  source = "terraform-aws-modules/ecs/aws"
+  source = "terraform-aws-modules/ecs"
```

Comment:
```suggestion
  source = "terraform-aws-modules/ecs/aws"
```

This is in many places throughout the docs.

Reply:
Updated in https://github.com/terraform-aws-modules/terraform-aws-ecs/pull/76/commits/c6fef3b1bb52c658ce3cd0a120a38c6e17bef170